mudlet-mapper.xml

```diff
@@ -3296,6 +3296,47 @@ end</script>
                     </regexCodePropertyList>
                 </Trigger>
             </TriggerGroup>
+            <Trigger isActive="yes" isFolder="no" isTempTrigger="no" isMultiline="yes" isPerlSlashGOption="no" isColorizerTrigger="no" isFilterTrigger="no" isSoundTrigger="no" isColorTrigger="no" isColorTriggerFg="no" isColorTriggerBg="no">
+                <name>City has orb of confinement</name>
+                <script>local cityIDs = {
+	49,		-- Ashtan
```

Comment:
Does this work both with crowdmap and the IRE map?

Reply:
I would guess not with the IRE map as I imagine it doesn't use the same area ID. I'll look into something to do with the gmcp info too

Reply:
Can anyone provide me with the IRE map area ID for Targ (enemied!)

Reply:
I can't right now, but if you download the IRE map and call the area list function, it would be there

Reply:
I've added in the necessary to get it to work with different area IDs for Targossas dependent on mpp.settings.crowdmap